jobs/build/sweep/Jenkinsfile

```
@@ -75,34 +83,53 @@ node {
         return
     }
 
-    currentBuild.description = "Repairing state and sweeping new bugs.\n"
     def (major, minor) = commonlib.extractMajorMinorVersionNumbers(version)
-    def kinds = major >= 4 ? ["image", "extras"] : ["rpm"]
 
-    stage("Repair bug state") {
-        currentBuild.description += "* Moving attached bugs in MODIFIED state to ON_QA...\n"
-        for (kind in kinds) {
+    currentBuild.description = "Repairing state and sweeping new bugs.\n"
```

There is no separate "repairing state" now so I'd just say:
```suggestion
    currentBuild.description = "Sweeping bugs.<br/>"
```
(Output should be HTML now)